src/Forms/Shared/Samples/Geoprocessing/AnalyzeHotspots/AnalyzeHotspots.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Tasks;
+using Esri.ArcGISRuntime.Tasks.Geoprocessing;
+using System;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace ArcGISRuntimeXamarin.Samples.AnalyzeHotspots
+{
+    public partial class AnalyzeHotspots : ContentPage
+    {
+
+        // Url for the geoprocessing service
+        private const string _hotspotUrl =
+            "http://sampleserver6.arcgisonline.com/arcgis/rest/services/911CallsHotspot/GPServer/911%20Calls%20Hotspot";
```

Comment:
Any url used in an iOS sample is required to be https. Or if there is no https equivalent available then the domain of the url must be added to the info.plist